tldw_Server_API/app/core/http_client.py

```
@@ -261,6 +273,29 @@ def _redact_headers(h: Optional[Dict[str, str]]) -> Dict[str, str]:
+    if ae and str(backend).lower() == "httpx":
+        try:
+            tokens = [t.strip() for t in ae.split(',') if t.strip()]
+            tokens = [t for t in tokens if t.lower() != 'zstd']
+            if tokens:
+                hdrs["Accept-Encoding"] = ", ".join(tokens)
+            else:
+                # If all tokens removed, drop the header
+                hdrs.pop("Accept-Encoding", None)
+        except Exception:
+            # Best-effort: if parsing fails, leave headers as-is
+            pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)